scripts/build-minified.js

```diff
@@ -9,7 +9,7 @@ const globalName = "oddjs"
 console.log("📦 Bundling & minifying...")
 
 await esbuild.build({
-  entryPoints: ["src/index.ts"],
+  entryPoints: ["src/index.ts", "src/compositions/fission.ts"],
```

Comment:
I wish I could use a glob here or point to a directory 🤔  We should probably come back to this script later when add more compositions and then use Node to list the files in that directory.